api/src/model/Recipe.ts

```diff
@@ -0,0 +1,5 @@
+export type Recipe = {
+    recipeId: string,
+    userId: string,
+    description: string
```

Comment:
Nit: we should make improve typing here and create wrapper types for recipeId, userId, etc. It would be very easy to accidentally pass userId when we mean to send recipeId since they are both strings